akka-stream/src/main/scala/akka/stream/ActorMaterializer.scala

```diff
@@ -308,6 +313,9 @@ final class ActorMaterializerSettings(
     })
   }
 
+  def withFuzzing(enable: Boolean): ActorMaterializerSettings =
+    copy(fuzzingMode = enable)
+
   def withDebugLogging(enable: Boolean): ActorMaterializerSettings =
     copy(debugLogging = enable)
```

Comment:
Could we make a quick boyscout and only copy if the new setting deviates from the current setting? (goes for all the `with` methods)